crates/store/src/genesis.rs

```diff
@@ -59,7 +58,7 @@ impl GenesisState {
             MmrPeaks::new(0, Vec::new()).unwrap().hash_peaks(),
             account_smt.root(),
             Smt::default().root(),
-            *EmptySubtreeRoots::entry(NOTE_LEAF_DEPTH, 0),
+            *EmptySubtreeRoots::entry(BLOCK_NOTE_TREE_DEPTH, 0),
```

Comment:
Side note: we should consider new-typing the header types. They're all `Digest` / `u32` which is a bit dangerous.